tests/kani/AsyncAwait/main.rs

```diff
@@ -0,0 +1,51 @@
+// Copyright Kani Contributors
+// SPDX-License-Identifier: Apache-2.0 OR MIT
+//
+// compile-flags: --edition 2018
+
+use std::{
+    future::Future,
+    pin::Pin,
+    task::{Context, RawWaker, RawWakerVTable, Waker},
+};
+
+#[kani::proof]
+#[kani::unwind(10)]
+fn main() {
```

Comment:
nit: Can you please use a more informative name instead of `main`.

Reply:
Done